app.js

```diff
@@ -428,8 +428,8 @@
         operations.push(this.buildPatchToAddWorkItemField("System.AreaId", areaId));
       }
 
-      if (this.hasFieldDefined(workItemType, "Microsoft.VSTS.Common.Severity") && $modal.find('.severity').val()) {
-        operations.push(this.buildPatchToAddWorkItemField("Microsoft.VSTS.Common.Severity", $modal.find('.severity').val()));
+      if (this.hasFieldDefined(workItemType, "Microsoft.VSTS.Common.Severity") && $modal.find('#severity').val()) {
+        operations.push(this.buildPatchToAddWorkItemField("Microsoft.VSTS.Common.Severity", $modal.find('#severity').val()));
```

Comment:
Good catch. I'd rather to add the "severity" class to the element since Zendesk considers a best practice to identify page elements by class rather than ID. 

https://github.com/zendesk/demo_apps/blob/master/Bestpractices.md

> Identify your page elements by class rather than by ID. Zendesk will replace your IDs, so if they are referenced in app.js they won't work. Giving elements unique classes is a workaround.